routes/web.php

```diff
@@ -255,34 +255,36 @@
     Route::group(['prefix' => 'v2'], function () {
         Route::group(['prefix' => 'chat'], function () {
             //  GET /api/v2/chat/channels
-            Route::get('channels', ['uses' => 'ChatController@channels']);
+            Route::get('channels', 'ChatController@channels');
             //  GET /api/v2/chat/messages
-            Route::get('messages', ['uses' => 'ChatController@messages']);
+            Route::get('messages', 'ChatController@messages');
             //  GET /api/v2/chat/messages/private
-            Route::get('messages/private', ['uses' => 'ChatController@privateMessages']);
+            Route::get('messages/private', 'ChatController@privateMessages');
             // POST /api/v2/chat/messages/new
-            Route::post('messages', ['uses' => 'ChatController@postMessage']);
+            Route::post('messages', 'ChatController@postMessage');
         });
 
         Route::resource('rooms', 'RoomsController', ['only' => ['show']]);
 
         Route::group(['prefix' => 'beatmapsets'], function () {
-            Route::get('favourites', ['uses' => 'BeatmapsetsController@favourites']);     //  GET /api/v2/beatmapsets/favourites
+            Route::get('favourites', 'BeatmapsetsController@favourites');     //  GET /api/v2/beatmapsets/favourites
```

Comment:
might as well merge with the `// Beatmapsets` group below while at it?

Reply:
hrm I wonder if that endpoint should be nested under `users` or `me` instead?

Reply:
hrm right :confused:

Reply:
Actually that endpoint probably isn't even needed anymore, I'll make a new PR removing that stuff.